apps/extension/src/routes/popup/settings/settings-clear-cache.tsx

```diff
@@ -6,9 +6,30 @@ import { TrashGradientIcon } from '../../../icons/trash-gradient';
 import { ServicesMessage } from '@penumbra-zone/types/src/services';
 import { usePopupNav } from '../../../utils/navigate';
 import { PopupPath } from '../paths';
+import { useStore } from '../../../state';
+import { useState } from 'react';
 
-export const SettingsClearCache = () => {
+const useCacheClear = () => {
   const navigate = usePopupNav();
+  const [loading, setLoading] = useState(false);
+
+  const handleCacheClear = () => {
+    setLoading(true);
+
+    void (async function () {
+      await chrome.runtime.sendMessage(ServicesMessage.ClearCache);
+      useStore.setState(state => {
+        state.network.fullSyncHeight = undefined;
+      });
```

Comment:
Two things happening here:
- Awaiting cache clearing to finish
- resetting zustand sync height

Only after this do we navigate. Sometimes this can be slower, so added a loading state.